test/integration/src/client-helper.js

```diff
@@ -10,7 +10,7 @@ module.exports = {
     getTranscribedJobId: (jobList) => {
         var completedJobId;
         for(job of jobList) {
-            if(job.status === 'transcribed') {
+            if(job.status === 'transcribed' && job.type != 'stream') {
```

Comment:
Do we have status enums defined? Should we?

Reply:
We do not have a status enum defined. I can't remember what our reasoning was for that. I think it's a good idea